octoprint_mrbeam/static/js/messages.js

```diff
@@ -19,6 +19,11 @@ $(function () {
         self.notificationsHandled = false;
         self.messagesLoaded = false;
 
+        self.onStartupComplete = function () {
+            // Hide Messaging icon
+            $("li a#messages_nav_tab").hide();
```

Comment:
I assume that `onUserLoggedIn` is called before `onStartupComplete` in the boot order. I assume you tested it and it worked, so maybe practice beats theory. Still, I suggest to put the hiding in `onStartup`. For this command you do not the the viewmodel to be bound therefore onStartup should be fine.